frontend/src/components/FeatureAttention.vue

```diff
@@ -3,21 +3,27 @@
     :class="customClass"
     :style="`WARN`"
     :title="$t(`subscription.features.${featureKey}.title`)"
-    :description="description"
-    :action-text="
-      subscriptionStore.canTrial
-        ? $t('subscription.start-n-days-trial', {
-            days: subscriptionStore.trialingDays,
-          })
-        : $t('subscription.upgrade')
-    "
+    :description="`${description}\n${$t(
+      'subscription.required-plan-with-trial',
+      {
+        requiredPlan: $t(
+          `subscription.plan.${planTypeToString(requiredPlan)}.title`
+        ),
+        startTrial: subscriptionStore.canUpgradeTrial
+          ? $t('subscription.upgrade-trial').toLowerCase()
+          : $t('subscription.trial-for-days', {
+              days: subscriptionStore.trialingDays,
+            }).toLowerCase(),
+      }
+    )}`"
```

Comment:
Maybe we can use a computed value here as same as the `actionText`. WDYT

Reply:
LGTM. Done